### PR TITLE
Usim branding fix (changed mentions of Usim in the tutorial to Unity Simulation)

### PR DESCRIPTION
**Pull request by mkamalza**
# Peer Review Information:
Information on any code, feature, documentation changes here

## Editor / Package versioning:
**Editor Version Target (i.e. 19.3, 20.1)**: 2019.3

## Dev Testing:
**Tests Added**: 

**Core Scenario Tested**: 

**At Risk Areas**: 

**Notes + Expectations**: 

## Checklist
- [ ] - Updated docs
- [ ] - Updated changelog

### CODE REVIEW

File: com.unity.perception/Documentation~/Tutorial/Phase3.md [hunk 1]
```diff
@@ -205,13 +205,13 @@ name        id        creation time         executions
                                               xBv3arj   Completed   2020-10-01 02:27:11    
 ```
 
-As seen above, each run has a name, an ID, a creation time, and a list of executions. Note that each "run" can have more than one "execution", as you can manually execute runs again using USim-CLI.
+As seen above, each run has a name, an ID, a creation time, and a list of executions. Note that each "run" can have more than one "execution", as you can manually execute runs again using USimCLI.
```
Comment (JonathanHUnity) [inline com.unity.perception/Documentation~/Tutorial/Phase3.md:208]: Should this be renamed as well?

**Review by JonathanHUnity — APPROVED** [review]
Ok, just one note

File: com.unity.perception/Documentation~/Tutorial/Phase3.md (hunk 1 above)
Comment (mkamalza) [reply com.unity.perception/Documentation~/Tutorial/Phase3.md:208]: USimCLI is the official name of the command-line interface, so I was hesitant to rename it in the tutorial.

Comment (mkamalza) [reply com.unity.perception/Documentation~/Tutorial/Phase3.md:208]: or maybe I'm wrong, it seems to only appear in the bundle itself

**Pull request merged**